components/producers/golang-nancy/main_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"testing"
 
 	v1 "github.com/ocurity/dracon/api/proto/v1"
```

Comment:
these imports are our own, they should be the last block, we should have first the standard library ones, then a block with the third parties, then our own